.github/workflows/build.yml

```diff
@@ -11,8 +12,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: |
-          which vcpkg
-          type vcpkg
+          find / -name vcpkg-init.sh
+      - run: |
           vcpkg activate
           csolution list packs -m -s GetStarted.csolution.yml > packs.txt
```

Comment:
I suggest to use
`cpackget add -a -f packs.txt`
to ensure that licenses don't require to be accepted interactively